packages/chart/package.json

```diff
@@ -38,12 +39,12 @@
     "react-plotly.js": "^2.4.0"
   },
   "peerDependencies": {
-    "@deephaven/components": "^0.x",
-    "@deephaven/icons": "^0.x",
-    "@deephaven/iris-grid": "^0.x",
-    "@deephaven/jsapi-shim": "^0.x",
-    "@deephaven/log": "^0.x",
-    "@deephaven/redux": "^0.x",
+    "@deephaven/components": "^0.3.1",
+    "@deephaven/icons": "^0.3.1",
+    "@deephaven/iris-grid": "^0.3.1",
+    "@deephaven/jsapi-shim": "^0.3.1",
+    "@deephaven/log": "^0.3.1",
+    "@deephaven/redux": "^0.3.1",
```

Comment:
I mean technically, they shouldn't _need_ to be bumped since these packages were compatible with the previous versions, since it was working with the old version... but yea I'm not sure if lerna can update all of these automatically.